doc/README.txt

```diff
@@ -1,11 +1,45 @@
 maptlotlib documentation
 ========================
 
+
+Building the documentation
+--------------------------
+
+A list of dependencies can be found in ../doc-requirements.txt.
+
+All of these dependencies can be installed through pip::
+
+  pip install sphinx numpydoc ipython mock colorspacious
```

Comment:
Yeah… I'd just really really really don't like giving (or running) commands that are not explicit… But I'll update this to use the doc-requirements.txt